src/granite/association_collection.cr

```diff
@@ -11,26 +11,23 @@ class Granite::AssociationCollection(Owner, Target)
     )
   end
 
-  def find_by(field : String | Symbol, value)
+  def find_by(**args)
     Target.first(
-      [query, "AND #{Target.table_name}.#{field} = ?"].join(" "),
-      [owner.id, value]
+      "#{query} AND #{args.map { |arg| "#{Target.table_name}.#{arg} = ?" }.join(" AND ")}",
```

Comment:
Should this table name and field names be quoted as well?

Reply:
Thanks